core/src/main/scala/akka/persistence/jdbc/util/PluginVersionChecker.scala

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (C) 2014 - 2019 Dennis Vriend <https://github.com/dnvriend>
+ * Copyright (C) 2019 - 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.persistence.jdbc.util
+
+object PluginVersionChecker {
+  def check() = 
+    try {
+      Class.forName("akka.persistence.jdbc.util.DefaultSlickDatabaseProvider")
```

Comment:
We check for `akka.persistence.jdbc.util.DefaultSlickDatabaseProvider` that only existing 3.5.0 or prior.